sass/responsive/_mobile.scss

```diff
@@ -1369,7 +1369,7 @@
 
     .search-bar__container {
         @include flex(0 0 50px);
-        background: $primary-color;
+        background: var(--sidebar-header-bg);
```

Comment:
@jgilliam17, this affects the background colour for the search bar header in mobile.